daemon/src/routes_maker.rs

```diff
@@ -106,3 +110,32 @@ pub fn get_health_check() {}
 
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
 struct RetrieveCurrentOrder;
+
+#[derive(RustEmbed)]
+#[folder = "../frontend/dist/maker"]
+struct Asset;
+
+#[rocket::get("/assets/<file..>")]
+pub fn dist<'r>(file: PathBuf) -> impl Responder<'r, 'static> {
+    let filename = format!("assets/{}", file.display().to_string());
+    Asset::get(&filename).map_or_else(
+        || Err(Status::NotFound),
+        |d| {
+            let ext = file
+                .as_path()
+                .extension()
+                .and_then(OsStr::to_str)
+                .ok_or_else(|| Status::new(400))?;
+            let content_type = ContentType::from_extension(ext).ok_or_else(|| Status::new(400))?;
+            Ok((content_type, d.data))
+        },
+    )
+}
```

Comment:
These are exactly the same for both right?

Can we have a shared implementation that is generic over the `RustEmbed` trait and hence allows to plugin in the respective `Asset` struct?

Reply:
I gave it a try in : [`28cc157` (#95)](https://github.com/comit-network/hermes/pull/95/commits/28cc157cee7da73cbe9a17ca2bbe59b876d019dc)